site/docs/reference/Connectors/capture-connectors/gcs.md

```diff
@@ -73,4 +73,35 @@ captures:
 
 Your capture definition may be more complex, with additional bindings for different GCS prefixes within the same bucket.
 
-[Learn more about capture definitions.](../../../concepts/captures.md#pull-captures)
\ No newline at end of file
+[Learn more about capture definitions.](../../../concepts/captures.md#pull-captures)
+
+### Configure Google service account impersonation
+
+As part of your Google IAM management, you may have configured one service account to [impersonate another service account](https://cloud.google.com/iam/docs/impersonating-service-accounts).
+This is useful when you want to easily control multiple service accounts with only one set of keys.
+
+You can configure this authorization model for a GCS capture in Flow using the GitOps workflow.
+To do so, you'll enable sops encryption and impersonate the target account with JSON credentials.
+
+Before you begin, make sure you're familiar with [how to encrypt credentials in Flow using sops](./../../../concepts/connectors.md#protecting-secrets).
+
+Use the following sample as a guide to add the credentials JSON to the capture's endpoint configuration.
+The sample uses the [encrypted suffix feature](../../../concepts/connectors.md#example-protect-portions-of-a-configuration) of sops to encrypt only the sensitive credentials, but you may choose to encrypt the entire configuration.
+
+```yaml
+config:
+  bucket: <bucket-name>
+  googleCredentials_sops:
+    # URL containing the account to impersonate and the associated project
+    service_account_impersonation_url: https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/<target-account>@<project>.iam.gserviceaccount.com:generateAccessToken
+    # Credentials for the account that has been configured to impersonate the target
+    source_credentials:
+        client_email: <origin-account>@<anotherproject>.iam.gserviceaccount.com
+        client_id: ...
```

Comment:
I'd snip out all the fields where I just wrote `...`, because that's not really an exhaustive list and the idea that I'm trying to convey there was just "copy-paste the same credentials JSON/YAML you would normally use into the `source_credentials` property".